app/assets/javascripts/discourse/app/routes/user-activity-bookmarks.js

```diff
@@ -9,6 +9,11 @@ export default DiscourseRoute.extend({
     q: { refreshModel: true },
   },
 
+  setupController(controller) {
+    this._super(...arguments);
+    controller.set("loading", false);
```

Comment:
I don't think this is working as intended. `setupController` is called before the template is rendered so this is equivalent to setting a default value of `false` for the controller which is the same default previously.

Reply:
I think what you want to be doing here is to set the controller's `loading` prop in the router's `model` hook as that is where the async loading actually happens.

Reply:
> I think what you want to be doing here is to set the controller's loading prop in the router's model hook as that is where the async loading actually happens.

I tried that and it did not work at all, what I have done here is the only thing that consistently showed the spinner.

Reply:
@tgxworld Hmm tried it again and now it works... maybe the `loading` action in the route was breaking it?? Anyway pushed it up now

Reply:
> so this is equivalent to setting a default value of false for the controller which is the same as the default previously.

O sorry this is wrong because you called `super` first so that would have set up the controller :thinking: 

Strange I tried it locally and I don't see the loading spinner.

Reply:
Anyway it's working the other way now :shrug: